spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -131,7 +132,7 @@ def __init__(self, parent, id_,
         self.options_button = options_button
         self.history = []
         self.allow_rename = True
-        self.stderr_dir = None
+        self.std_dir = std_dir

```

Comment:
```suggestion
        self.stderr_dir = None
```

Please revert this change because it's not necessary.